app/views/results/jobHistoryResults.scala.html

```diff
@@ -81,7 +81,7 @@ <h4><a href="@jobDefPair.getId()">Job History</a></h4>
                       <!-- Table Tooltip Content -->
                       <div id="tooltip-div">
                         <p><b>@appHeuristicResult.yarnAppResult.id</b></p>
-                        <p><b>@appHeuristicResult.heuristicName</b></p>
```

Comment:
if you want to remove the @attempt* string from root webpage and then on details jobs top description you must do it here
```
diff --git a/app/views/results/jobDetails.scala.html b/app/views/results/jobDetails.scala.html
index 81566a3..9fa8cdc 100644
--- a/app/views/results/jobDetails.scala.html
+++ b/app/views/results/jobDetails.scala.html
@@ -90,7 +90,7 @@
         <br>
         <div>
           @for(yarnAppHeuristicResult <- result.yarnAppHeuristicResults) {
-            <a href="#@yarnAppHeuristicResult.heuristicName.replace(" ", "")" class="label label-@yarnAppHeuristicResult.severity.getBootstrapColor">@yarnAppHeuristicResult.heuristicName</a>
+            <a href="#@yarnAppHeuristicResult.heuristicName.replace(" ", "")" class="label label-@yarnAppHeuristicResult.severity.getBootstrapColor">@controllers.Application.trimHeuristicName(yarnAppHeuristicResult.heuristicName)</a>
           }
         </div>
       </p>
diff --git a/app/views/tags/jobSummary.scala.html b/app/views/tags/jobSummary.scala.html
index 98a01df..a4bb5d3 100644
--- a/app/views/tags/jobSummary.scala.html
+++ b/app/views/tags/jobSummary.scala.html
@@ -26,7 +26,7 @@
 @tags.jobHeader(result)
 <p>
   @for(appHeuristicResult <- result.yarnAppHeuristicResults) {
-    <span class="label label-@appHeuristicResult.severity.getBootstrapColor">@appHeuristicResult.heuristicName</span>
+    <span class="label label-@appHeuristicResult.severity.getBootstrapColor">@controllers.Application.trimHeuristicName(appHeuristicResult.heuristicName)</span>
   }
 </p>

```

Reply:
Thanks, I have changed this way